bots/mainbot/castle.js

```diff
@@ -101,31 +113,26 @@ castle.takeTurn = (self) => {
 					self.log('Building a prophet at ' + (self.me.x+1) + ',' + (self.me.y+1));
 					 return self.buildUnit(SPECS.PROPHET, d.x, d.y);
 				}
-				var d = getBuildDir(self.me);
-				if (!(d === undefined)){
```

Comment:
Did you mean to stop building crusaders, or was this just for your testing?

Reply:
Stopped building them because I have their movement commented out and they were using resources. The castle rework adds the different strategies including crusader stuff.